### PR TITLE
Release/2.0.0

### PR DESCRIPTION
**Pull request by Xuanwo**

### CODE REVIEW

File: dist/qingstor-sdk.js [hunk 1]
```diff
@@ -3241,7 +3241,7 @@ exports.unpack = Unpacker;
 
 'use strict';
 
-global.version = '2.0.0-beta.6';
```
Comment (prettyxw) [inline dist/qingstor-sdk.js]: The `dist` directory should be removed from the repository.

File: lib/version.js [hunk 2]
```diff
@@ -16,4 +16,4 @@
 
 'use strict';
 
-global.version = '2.0.0-beta.6';
\ No newline at end of file
+global.version = '2.0.0';
```
Comment (prettyxw) [inline lib/version.js]: Not newline at the end of file.

File: test/unpack.test.js [hunk 3]
```diff
@@ -1,41 +0,0 @@
-// +-------------------------------------------------------------------------
```
Comment (prettyxw) [inline test/unpack.test.js]: Why remove the test?

File: lib/version.js (hunk 2 above)
Comment (Xuanwo) [inline lib/version.js]: Fixed.

File: dist/qingstor-sdk.js (hunk 1 above)
Comment (Xuanwo) [inline dist/qingstor-sdk.js]: Removed.

File: .gitignore [hunk 4]
```diff
@@ -38,4 +37,6 @@ jspm_packages
 # Optional REPL history
 .node_repl_history
 
-test_config.yaml
\ No newline at end of file
+test_config.yaml
+
+dist/
```
Comment (prettyxw) [inline .gitignore]: No newline at the end of file.

Comment (Xuanwo) [inline .gitignore]: Fixed.

**Pull request merged**